### DISCUSSION TITLE
GiuaReg v2.0

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#397] [Richieste di aiuto]
Buongiorno, vorrei sapere se l'app   "GiuaReg v2.0" è utilizzabile anche dal personale della mia scuola o e un'app personalizzata solo per un istituto specifico?

**Comment by trinko** [comment]
No, questa è personalizzata.
Quando sarà possibile, devo aggiornarla per gestire anche lo SPID: in quella occasione la potrò rendere di uso generale.
